### ISSUE TITLE
6-01 Sampling procedure (new codelist)

### ISSUE THREAD

**Issue by joergklausen** [wmo-im/wmds#111]
**Branch**
https://github.com/wmo-im/wmds/tree/issue111

**Summary and Purpose**
[your arguments why the existing situation is not satisfying]

**Stakeholder(s)**
[include emails or handles as relevant]

**Proposal**
[your specific proposal to resolve the issue]

**Reason**
[your reasoning why the proposal resolves the issue]

***************************
**original comment**
[6-01_t_go_sampl_procedure_ref.txt](https://github.com/wmo-im/wmds/files/3334152/6-01_t_go_sampl_procedure_ref.txt)

**Comment by LuisaIckes** [comment]
To make it easier to look at the table coming from OSCAR click [here](https://github.com/wmo-im/wmds/blob/issue111/tables_en/6-01.csv) (note that name, definition, notation are not in the usual order).

**Comment by LuisaIckes** [comment]
@wmo-im/tt-wmd : I inivite everyone to comment on this new table!

**Comment by LuisaIckes** [comment]
05.11.2019: Finally accepted in teleconference.

**Comment by amilan17** [comment]
This was skipped in the last FT cycle. Reopening to include in FT-2021-1.

**Comment by fstuerzl** [comment]
Corrections in column "description":

The description of the entry "PM1 (heated)" should say 
"sampling of particles with size cut-off at 1 µm and at elevated temperature and therefore reduced relative humidity (normally <40% RH)" 
instead of 
"sampling of particles with size cut-off at 1 µm and at elevated temperature and therefore reduced relative humidity (normally <40% RH) at elevated temperature and therefore reduced relative humidity (normally <40% RH)". The same issue occurs in the descriptions of "PM2.5 (heated)" and "PM10 (heated)". 

[6-01_t_go_sampl_procedure_ref.txt](https://github.com/wmo-im/wmds/files/5582108/6-01_t_go_sampl_procedure_ref.txt)

**Comment by fstuerzl** [comment]
This issue depends on #173  as described in the WG ACV wiki page  https://github.com/wmo-im/wmds/wiki/wg-acv  and needs a review.

**Comment by joergklausen** [comment]
Pending further developments from WG-ACV.

**Comment by joergklausen** [comment]
Look at this in the context of #112

**Comment by joergklausen** [comment]
No longer relevant
